extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/search-configuration.xml

```diff
@@ -60,7 +60,7 @@
               <string>people</string>
             </field>
             <field name="uri">
-              <string><![CDATA[/portal/rest/v1/social/users?q={keyword}&limit={limit}&expand=all,spacesCount,relationshipStatus,connectionsCount]]></string>
```

Comment:
As I understand this is the request launched with the search app. 
you add the flag excludeExternal=true, so the request will be modified. 
But, what happens if the internal user get the request in chrome dev tool, and use excludeExternal=false ? or remove this attribute ?

Reply:
it gets a full search result (with external users), I didn't change the [getUsers()](https://github.com/Meeds-io/social/blob/c7faaddf21e50eece9e0d350a986fc48d5a57f5c/component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java#L262) rest method, because it's a generic method and used in other places.

Reply:
So, if a internal user get the rest service without excludeExternal, he will be able to read all users, including external. 
This should be ask to Sameh and Samuel, because I'm not sure that it is the wanted behaviour

Reply:
In the web service, if excludeExternal==false and user is not admin, then the service returns 403, so it is ok with this modification